docs/sources/getting-started/getting-started.md

```diff
@@ -1,58 +1,59 @@
 
-## Log in for the first time 
+## Step 2: Log in
 
-1. Open your web browser and go to http://localhost:3000/. `3000` is the default HTTP port that Grafana listens to if you haven’t configured a different port.
-1. On the login page, type `admin` for the username and password.
-1. Change your password. 
+To login to Grafana for the first time:
 
-> **Note:** We strongly encourage you to follow Grafana best practices and change the default administrator password. Don't forget to record your credentials!
+1. Open your web browser and go to http://localhost:3000/. The default HTTP port that Grafana listens to is `3000` unless you have configured a different port.
+1. On the login page, enter `admin` for username and password.
+1. Click **Log In**.
+1. If login is successful, you will see a prompt to change the password.
+1. Click **OK** on the prompt, then change your password.
 
-## Create a dashboard
+> **Note:** We strongly recommend that you follow Grafana's best practices and change the default administrator password. Don't forget to record your credentials!
 
-1. Click **New dashboard**.
-1. Click **Add Query**. Grafana creates a basic graph panel with the Random Walk scenario.
-1. Save your dashboard. Click the **Save dashboard** icon in the top corner of the screen.
+## Step 3: Create a dashboard
 
- Congratulations, you have gotten started with Grafana! You have a dashboard and are displaying results. Feel free to experiment with what you have built, continue on to add another data source, or explore [Next steps](#next-steps).
+To create your first dashboard:
 
-## Next steps
+1. Click the **+** icon on the left panel, select **Create Dashboard**, then click **Add new panel**.
+1. In the New Dashbard/Edit Panel view, go to the **Query** tab.
+1. Configure your [query]({{< relref "../panels/queries.md" >}}) by selecting ``-- Grafana --`` from the [data source selector]({{< relref "../panels/queries.md/#data-source-selector" >}}). This generates the Random Walk dashboard.
+1. Click the  **Save** icon in the top right corner of your screen to save the dashboard.
+1. Add a descriptive name, and click **Save**.
 
-Different user types will have different interests. Some suggestions are listed below, or refer to [What is Grafana?]({{< relref "what-is-grafana.md" >}}) for a general overview of Grafana features.
+ Congratulations, you have created your first dashboard and it is displaying results.
 
-### All users
+## Next steps
 
-All users might want to learn about:
+ Continue to experiment with what you have built, try the [explore workflow]({{< relref "../explore/index.md" >}}) or another visualization feature. Refer to [Data sources]({{< relref "../features/datasources/data-sources.md" >}}) for a list of supported data sources and instructions on how to [add a data source]({{< relref "../features/datasources/add-a-data-source.md" >}}). The following topics will be of interest to you:  
```

Comment:
```suggestion
 Continue to experiment with what you have built, try the [Explore workflow]({{< relref "../explore/index.md" >}}) or another visualization feature. Refer to [Data sources]({{< relref "../features/datasources/data-sources.md" >}}) for a list of supported data sources and instructions on how to [add a data source]({{< relref "../features/datasources/add-a-data-source.md" >}}). The following topics will be of interest to you:  
```

Reply:
I'm not sure I like the two links removed from the list and inserted into the long paragraph. That might be a little hard to read, and as a user I would either wonder why the links in there are treated differently. Or I might skip past the paragraph down to the list and miss them.

Reply:
Both are links to specific downloads, that is why I have listed them separately.